build.xml

```diff
@@ -811,6 +811,16 @@ To get started using Eclipse, execute "./build.py build-dev" and import the top-
             </copy>
     </target>
 
+    <target name="copy-bftools" depends="init">
+            <copy todir="${dist.dir}/bin">
+                <fileset dir="${bioformats.comp}/tools"/>
+            </copy>
+            <chmod dir="${dist.dir}/bin" perm="a+x" includes="*"/>
+            <copy todir="${dist.dir}/bin">
+                <fileset file="${bioformats.comp}/artifacts/bioformats_package.jar"/>
```

Comment:
I wonder if it's not possible to fix `config.sh` at the low-level Bio-Formats level to reuse the JARs already copied under `lib/server`. I will have a go at this if that makes sense.